examples/webgl2_mrt.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - Multiple Render Targets</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #000;
+				color: #fff;
+				margin: 0px;
+				overflow: hidden;
+			}
+			#info {
+				color: #fff;
+				position: absolute;
+				top: 10px;
+				width: 100%;
+				text-align: center;
+				display:block;
+			}
+			#info a, .button { color: #f00; font-weight: bold; text-decoration: underline; cursor: pointer }
+		</style>
+
+		<!-- Write to G-Buffer -->
+		<script id="gbuffer-vert" type="x-shader/x-vertex">
+			#version 300 es
```

Comment:
Please remove the version qualifiers from the shaders and set the `glslVersion` material property to `THREE.GLSL3`. Similar to other WebGL 2 examples like: 

https://threejs.org/examples/#webgl2_buffergeometry_attributes_integer

We actually don't want that users manage the version qualifiers by themselves. Instead they should declare what GLSL version they use when the material is created.